src/lib/crypto/kimchi_backend/gadgets/ec_group.ml

```diff
@@ -39,16 +39,14 @@ let is_on_curve_bignum_point (curve : Curve_params.t)
  *     L and R are not O (the point at infinity)
  *
  *   External checks: (not counting inputs and output)
- *     Bound checks:          6
- *     Multi-range-checks:    3
+ *     Bound checks:          12
+ *     Multi-range-checks:    12
  *     Compact-range-checks:  3
- *     Total range-checks:   12
+ *     Total range-checks:    93
  *
  *   Rows: (not counting inputs/outputs and constants)
- *     Group addition:     13
- *     Bound additions:    12
- *     Multi-range-checks: 48
- *     Total:              73
+ *     Group addition:  35
+ *     External checks: 135
```

Comment:
Oh, it does get more complex here though 😢 

What's the main reason for this? I saw the ec operations were actually more efficient now, right?

Reply:
This increase is because the ff additions now require the multi-range-checks Gregor requested.

Reply:
Also, note those values are counting slightly different things than previously.